packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -221,7 +237,7 @@ const ImageDialog = ({ handleClose }) => {
 
   return (
     <MediaLibraryDialog
-      allowedTypes={[ALLOWED_MEDIA_TYPE]}
+      allowedTypes={['images']}
```

Comment:
Why do you think to not use constant here?

Reply:
It doesn't seem needed to me as it's only used here, and we'll never allow anything other than image